packages/actor-abstract-path/lib/ActorAbstractPath.ts

```diff
@@ -83,6 +82,7 @@ export abstract class ActorAbstractPath extends ActorQueryOperationTypedMediated
     predicate: Algebra.PropertyPathSymbol,
     graph: RDF.Variable,
     context: IActionContext,
+    BF: BindingsFactory,
```

Comment:
This is a nit-picky comment, but could we write out the parameter name as `bindingsFactory`? Just to be consistent with the coding style of other methods params.
(this comment also applies to the rest of this PR)